packages/devtools-kit/src/ctx/hook.ts

```diff
@@ -295,6 +298,25 @@ export function createDevToolsCtxHooks() {
           resolve()
         }, DevToolsV6PluginAPIHookKeys.GET_INSPECTOR_STATE)
       })
+
+      // Expose instance data to window
+      // Copied from https://github.com/vuejs/devtools/blob/f03590025b0b4910cf539531c91384be51a8f8fa/packages/app-backend-core/src/component.ts#L57-L72
+      const instance = getComponentInstance(activeAppRecord.value, _payload.nodeId)
+      if (typeof window !== 'undefined' && instance) {
+        const win = window as any
+        win.$vm = instance
+
+        // $vm0, $vm1, $vm2, ...
+        if ($vmQueue[0] !== instance) {
```

Comment:
The instances are saved as `$vm0, $vm1, $vm2....`

But the user cannot know the correspondence.

Reply:
Maybe we can show the correspondence like `<Component /> == $vm2`, inspired from Chrome devtool.

![Screenshot 2024-06-10 at 15 44 42](https://github.com/vuejs/devtools-next/assets/22590005/c00d8213-e2f3-4846-a5f2-b4a0bbae4525)

Reply:
That sounds great, but implementing this might be challenging since the expose logic is within the sendInspectorState hook in the core package. I’m unsure how to send a signal to notify the client side. 

We can address this in a future PR.